guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md

```diff
@@ -182,9 +183,9 @@ visualize_dataset(train_dataset, title="After RandAugment")
 ```
 
 
-
-![png](/img/guides/cut_mix_mix_up_and_rand_augment/cut_mix_mix_up_and_rand_augment_15_0.png)
-
+    
+![png](../guides/img/cut_mix_mix_up_and_rand_augment/cut_mix_mix_up_and_rand_augment_15_0.png)
```

Comment:
I don't know how you generate the guides, but the image links have an incorrect path -- it should be `/img/guides/`.

Reply:
Please manually change the links and then use `python autogen.py make` / `serve` to check that it worked. I'll do some local tests to see what's wrong with the generation afterwards...

Reply:
We could always hard code a string replace operation in autogen too if we want to go the low tech route.